pkg/cloudprovider/providers/azure/azure_storage.go

```diff
@@ -27,6 +27,7 @@ import (
 const (
 	defaultStorageAccountType  = string(storage.StandardLRS)
 	fileShareAccountNamePrefix = "f"
+	azureDiskAccountNamePrefix = "dk"
```

Comment:
why not 'd', if 'k' has special meaning consider add some comments here

Reply:
This is for support kind = Shared accounts. one letter `d` is not enough, and I also don't want to use `disk` it has 4 letters, so I use `dk` it's not a word, to differentiate customer self created accounts. A little balancing tricky ...